### ISSUE TITLE
Can't detect initial thread stack location - find_vma failed

### ISSUE THREAD

**Issue by aiisuika** [xeffyr/termux-extra-packages#31]
Whenever Java is run, the following warning appears:
```bash
OpenJDK 64-Bit Server VM warning: Can't detect initial thread stack location - find_vma failed
```
After reading about it, it's due to `/proc` problems on standard linux distros... My device does have Android Oreo (after a recent update) which would make sense as `/proc` is almost inaccessible, however before Oreo, the same warning would appear...
Java is functioning a-ok though, I can run things like the Android SDK Tools and Gradle.

One thing to note, 800% CPU is hit often, even running basic things, and arm64 Ubuntu's openjdk-whatever distro's don't go over 60% CPU, any idea what's happening here?

Current phone info:
```bash
System information: Linux localhost 4.4.13-13824581-QB18772733 #1 SMP PREEMPT Mon Jul 2 12:06:04 KST 2018 aarch64 Android
Termux-packages arch: aarch64
Android version: 8.0.0
Device manufacturer: samsung
Device model: SM-A530F
```

**Comment by xeffyr** [comment]
> Can't detect initial thread stack location - find_vma failed

That's normal and does not harm anything.

> One thing to note, 800% CPU is hit often, even running basic things

Thats why OpenJDK is not available officially for Termux. Sometimes it even may crash (reboot) device.

> and arm64 Ubuntu's openjdk-whatever distro's don't go over 60% CPU, any idea what's happening here?

This can be at least because OpenJDK is compiled against bionic. Another source of problem may be usage of `android` compilation target (it is just not possible to use `linux` target here).

**Comment by aiisuika** [comment]
> That's normal and does not harm anything.

Yeah noticed. Just curious as to why it appears.

> Sometimes it even may crash (reboot) device.

I belive it's a ulimit problem, with the amount of threads the jvm creates (running `:(){ :|: & };:` in bash will crash Android due to the amount of processes)

< This can be at least because OpenJDK is compiled against bionic

Yeah, openjdk is make for glibc. I suppose I'll try compiling it for glibc on Android.

**Comment by xeffyr** [comment]
> Just curious as to why it appears.

Just to note: `find_vma` error is not exist on ARM.

> I belive it's a ulimit problem

> with the amount of threads the jvm creates

Just checked JDK 9 on PC - same amount of processes.

> running :(){ :|: & };: in bash will crash Android

Fork bomb can easily crash anything (mobile device, PC, server, etc...) if process amount is not limited to reasonable amount. Unfortunately Google don't want to limit amount of processes on Android leaving users vulnerable to this kind of DoS.

On my Android device I have soft limit of 12972 (!!!) processes while on PC the limit is only 2048.

> I suppose I'll try compiling it for glibc on Android.

Having glibc for Termux will require patching it and having compiler built specially for 'glibc for Termux'.

**NOTE:** Even if glibc will be available for Termux, 99.5% of software built with glibc for Linux will not work (or not work properly) because Termux have a prefixed rootfs (root is not `/` but `$PREFIX`).
